content/blog/authors/shafiya/contents.lr

```diff
@@ -0,0 +1,12 @@
+username: shafiya
+---
+name: Shafiya Heena
+---
+md5_hashed_email: f49628fb50324d628379b469c7f008c6
```

Comment:
yeah the original picture is already of low resolution as I took it from a video.So I feel hash made it more low but I feel this would work though it is blur